src/script/interpreter.cpp

```diff
@@ -1286,6 +1286,91 @@ bool EvalScript(std::vector<valtype> &stack, const CScript &script,
                         std::reverse(data.begin(), data.end());
                     } break;
 
+                    case OP_RAWBITSHIFT: {
```

Comment:
Do you have any opinion about inlining these new things? These giant functions drive me a bit mad.

Reply:
I don't know. My C++ isn't great, I mainly imitate the existing style.

BSV has a function at the top of `interpreter.cpp` but it looks very yanky.

Reply:
Can leave the style alone for now. But having inline functions would be nice here and in the networking code at some point.

Reply:
Ideally, we'd inline all opcodes and then the interpreter is just
```
case OP_BLAH:
    if (!EvalOpBlah(...)) return false;
    break;
....
```

Reply:
Yeah. But you're right. Let's leave it alone for now to avoid backport issues.